src/env.rs

```diff
@@ -1,20 +1,27 @@
 use envy::Error::MissingValue;
 use serde::Deserialize;
 
+use crate::types::Network;
+
 #[derive(Deserialize, Debug)]
 pub struct Environment {
+    #[serde(default = "default_network")]
+    pub network_name: Network,
     #[serde(default = "default_blobscan_api_endpoint")]
     pub blobscan_api_endpoint: String,
     #[serde(default = "default_beacon_node_endpoint")]
     pub beacon_node_endpoint: String,
     #[serde(default = "default_execution_node_endpoint")]
     pub execution_node_endpoint: String,
     pub secret_key: String,
-    #[serde(default = "default_dencun_fork_slot")]
-    pub dencun_fork_slot: u32,
+    pub dencun_fork_slot: Option<u32>,
     pub sentry_dsn: Option<String>,
 }
 
+fn default_network() -> Network {
+    Network::Devnet
```

Comment:
I wonder if we should use one of the existing network as default. And once mainnet is activated default to that.

Reply:
We can set mainnet as default once the upgrade happens. Right now I can't think of any reason to prefer one testnet over the other 🤔